admin_guide/crictl.adoc

```diff
@@ -0,0 +1,466 @@
+{
+  "images": [
+    {
+      "id": "f9d5de0795395db6c50cb1ac82ebed1bd8eb3eefcebb1aa724e01239594e937b",
+      "repoTags": [
+        "docker.io/kubernetes/pause:latest"
+      ],
+      "repoDigests": [
+        "docker.io/kubernetes/pause@sha256:2088df8eb02f10aae012e6d4bc212cabb0ada93cb05f09e504af0c9811e0ca14"
+      ],
+      "size": "250665",
+      "username": ""
+    },
+    {
+      "id": "8c811b4aec35f259572d0f79207bc0678df4c736eeec50bc9fec37ed936a472a",
+      "repoTags": [
+        "docker.io/library/busybox:latest"
+      ],
+      "repoDigests": [
+        "docker.io/library/busybox@sha256:74f634b1bc1bd74535d5209589734efbd44a25f4e2dc96d78784576a3eb5b335"
+      ],
+      "size": "1362408",
+      "username": ""
+    },
+    {
+      "id": "49f7960eb7e4cb46f1a02c1f8174c6fac07ebf1eb6d8deffbcb5c695f1c9edd5",
+      "repoTags": [
+        "docker.io/library/centos:7"
+      ],
+      "repoDigests": [
+        "docker.io/library/centos@sha256:eed5b251b615d1e70b10bcec578d64e8aa839d2785c2ffd5424e472818c42755"
+      ],
+      "size": "208242862",
+      "username": ""
+    },
+    {
+      "id": "73a557ff177a50bab42b5e36c9c8ba4c9fa18f2fe7a74f04900856ab1648cb89",
+      "repoTags": [
+        "docker.io/library/httpd:2.4-alpine"
+      ],
+      "repoDigests": [
+        "docker.io/library/httpd@sha256:aee15196959ee6553c9e623cb9b78d7579dfd3c3cacf474bffbe9614ff8aaf43"
+      ],
+      "size": "96002744",
+      "username": ""
+    }
+  ]
+}
+----
+
+The `*crictl inspecti*` command lets you view the spec of an image.
+----
+# crictl inspecti docker.io/library/centos:7
+{
+  "status": {
+    "id": "49f7960eb7e4cb46f1a02c1f8174c6fac07ebf1eb6d8deffbcb5c695f1c9edd5",
+    "repoTags": [
+      "docker.io/library/centos:7"
+    ],
+    "repoDigests": [
+      "docker.io/library/centos@sha256:eed5b251b615d1e70b10bcec578d64e8aa839d2785c2ffd5424e472818c42755"
+    ],
+    "size": "208242862",
+    "username": ""
+  }
+}
+----
+
+==== Deleting Images
+We can delete images from the local image store using the `*crictl rmi*` command.
+----
+# crictl images
+IMAGE                        TAG                 IMAGE ID            SIZE
+docker.io/kubernetes/pause   latest              f9d5de0795395       251kB
+docker.io/library/busybox    latest              8c811b4aec35f       1.36MB
+docker.io/library/centos     7                   49f7960eb7e4c       208MB
+docker.io/library/httpd      2.4-alpine          73a557ff177a5       96MB
+
+# crictl rmi docker.io/library/httpd:2.4-alpine
+Deleted: docker.io/library/httpd:2.4-alpine
+
+# crictl images
+IMAGE                        TAG                 IMAGE ID            SIZE
+docker.io/kubernetes/pause   latest              f9d5de0795395       251kB
+docker.io/library/busybox    latest              8c811b4aec35f       1.36MB
+docker.io/library/centos     7                   49f7960eb7e4c       208MB
+----
+
+*crictl* helps you do everything you can to debug issues with your pods and containers!
+Have fun :D
```

Comment:
Should remove this line as it is expected to go into official docs :)

Reply:
Whoops, forgot to do that. Fixed